examples/main.go

```diff
@@ -40,20 +41,19 @@ func main() {
 	tracker := sp.InitTracker(
 		sp.RequireEmitter(emitter),
 		sp.OptionSubject(subject),
+		sp.OptionBase64Encode(true),
```

Comment:
Nit: just for setting up the experiment, wouldn't the gzip compression be more efficient in case we didn't encode to base 64? I think we lose some potential for compression by using base64 before gzip.